### DISCUSSION TITLE
Music Assistant features - service to map MASS buttons to automations

### DISCUSSION THREAD

**Discussion by abacao** [music-assistant/support#2300] [Feature requests and ideas]
Hello and thank you for the project.

Please consider exposing more information to Home Assistant

- having a service that could get the current track playing (last 3 or so).
- a way to invoke the player settings in a service to be able to map buttons such as "heart" to a remote
- use text to speech for announcements using MASS

**Comment by OzGav** [comment]
You can already get the current track playing from the attributes of the player

not exactly sure what you mean on your second point

You can already do TTS to MA players. See the docs on the player announcement service call page.
